packages/hadron-build/package.json

```diff
@@ -58,7 +58,7 @@
     "lodash": "^4.17.15",
     "moment": "^2.17.1",
     "mongodb-js-cli": "^0.0.3",
-    "mongodb-notary-service-client": "^0.5.0",
+    "@mongodb-js/mongodb-notary-service-client": "^0.5.0",
```

Comment:
Seems to be used in https://github.com/mongodb-js/compass/blob/170c704d3e22d335d5bf380dafe2b87569d7c4be/packages/hadron-build/lib/target.js#L12 but I don't see this file being updated in the patch

Reply:
Great catch! let me re-do the search and replace, i missed something